components/form/overview.md

```diff
@@ -212,19 +212,44 @@ The Blazor Form exposes multiple parameters that allow you to customize its layo
 
 ## Form Reference
 
-Use the Form Reference to get access to the `EditContext` object that the form will generate when you pass a `Model` to it. It could be useful to, for example, re-attach validation when you change the model - `FormReference.EditContext.AddDataAnnotationsValidation()`.
+Use the Form Reference to get access its `EditContext`. The Form generates this object, no matter if the component uses a `Model` or an `EditContext` parameter. You can validate the `EditContext` manually or re-attach validation when you change the model - `FormReference.EditContext.AddDataAnnotationsValidation()`.
```

Comment:
In the first sentence (Use the Form Reference to get access its `EditContext`.) we can add Use the Form Reference to get access `to` its EditContext`.